src/home_robot_hw/home_robot_hw/ros/camera.py

```diff
@@ -49,8 +48,11 @@ def valid_pc(self, xyz, rgb, depth):
         rgb = rgb.reshape(-1, 3)[mask]
         return xyz, rgb
 
-    def wait_for_image(self):
-        rate = rospy.Rate(10)
+    def wait_for_image(self) -> None:
+        """ Wait for image. Needs to be sort of slow, in order to make sure we give it time
+        to update the image in the backend."""
+        rospy.sleep(0.5)
+        rate = rospy.Rate(2)
```

Comment:
Use class parameters (like `StretchEnv.goal_time_tolerance`), or constants defined at top of file.

Reply:
+1 to informative constant name in file